src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -83,11 +86,24 @@ String getParcelableCode(Context context, String className, String classToExtend
 
   private void validateProperties(ProcessingEnvironment env,
                                   Map<String, ExecutableElement> properties) {
+    Types typeUtils = env.getTypeUtils();
+    Elements elementUtils = env.getElementUtils();
+    TypeMirror collection = typeUtils.getDeclaredType(
+        elementUtils.getTypeElement(Collection.class.getName()),
```

Comment:
Shouldn't this be more narrow at just `List`? The `Parcel` methods require a list so a `Set` or just `Collection` wouldn't be acceptable. Granted, most implementations of these types are likely to be `Serializable` themselves, but I don't think we can rely on that.